docs/typescript.md

```diff
@@ -54,3 +54,22 @@ the Functions Framework.
     Function: helloWorld
     URL: http://localhost:8080/
     ```
+
+## Deploying with gcloud CLI
+
+1. Adjust `main` field in `package.json` to point to the compiled javascript source. `gcloud` will inspect the `main` field to detect sources:
```

Comment:
Remove the sentence about gcloud to simplify. That sentence isn't technically correct. There is no inspection from the CLI.